<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>385</width>
    <height>237</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <family>Agency FB</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>筆刷大小</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="slider_value_lb">
         <property name="font">
          <font>
           <family>Agency FB</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel{
background:rgb(221, 221, 221)
}</string>
         </property>
         <property name="text">
          <string>5</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="pen_size_slider">
         <property name="minimum">
          <number>4</number>
         </property>
         <property name="maximum">
          <number>200</number>
         </property>
         <property name="value">
          <number>5</number>
         </property>
         <property name="sliderPosition">
          <number>5</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QComboBox" name="selected_algo_cbox">
       <property name="font">
        <font>
         <family>Agency FB</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <item>
        <property name="text">
         <string>Nothing</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Mean</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Medium</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>GaussianBlur</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Max</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Min</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Unsharp_mean</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Unsharp_gaussian</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Histogram equalization</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>Agency FB</family>
           <pointsize>14</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>FIilter 大小</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>Agency FB</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Width:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="width_sb">
         <property name="minimum">
          <number>3</number>
         </property>
         <property name="maximum">
          <number>7</number>
         </property>
         <property name="value">
          <number>5</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>Agency FB</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Height:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="height_sb">
         <property name="minimum">
          <number>3</number>
         </property>
         <property name="maximum">
          <number>7</number>
         </property>
         <property name="value">
          <number>5</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pen_size_slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>slider_value_lb</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>249</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>98</x>
     <y>22</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
